test/functional/apps/visualize/_region_map.js

```diff
@@ -45,7 +45,7 @@ export default function ({ getService, getPageObjects }) {
         })
         .then(function selectVectorMap() {
           log.debug('Vector Map = World Countries');
```

Comment:
Am I reading this wrong? Shouldn't this log be `Vector Map = USA States`?

Reply:
no, good catch, the log is wrong!